src/StyledElement.js

```diff
@@ -1,6 +1,6 @@
-import { BaseElement } from './BaseElement';
-import { supportsAdoptingStyleSheets, getShadowParentOrBody } from './util/DOMHelper';
-export { i18n } from './util/i18n';
+import { BaseElement } from './BaseElement.js';
```

Comment:
Has this been consistently wrong before or is it now ?
Is this covered by some sort of lint / prettier setting ?